debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/IncrementalSnapshotIT.java

```diff
@@ -49,8 +53,8 @@ public class IncrementalSnapshotIT extends AbstractIncrementalSnapshotTest<Postg
             + "CREATE TABLE s1.a4 (pk1 integer, pk2 integer, pk3 integer, pk4 integer, aa integer, PRIMARY KEY(pk1, pk2, pk3, pk4));"
             + "CREATE TABLE s1.a42 (pk1 integer, pk2 integer, pk3 integer, pk4 integer, aa integer);"
             + "CREATE TABLE s1.anumeric (pk numeric, aa integer, PRIMARY KEY(pk));"
-            + "CREATE TABLE s1.debezium_signal (id varchar(64), type varchar(32), data varchar(2048));"
-//            + "ALTER TABLE s1.debezium_signal REPLICA IDENTITY FULL;"
+            + "CREATE TABLE s1.debezium_signal (id varchar(64), type varchar(32), data varchar(2048), sno serial PRIMARY KEY);"
+            // + "ALTER TABLE s1.debezium_signal REPLICA IDENTITY FULL;"
```

Comment:
Can you please add a `// YB Note:` comment explaining why this line is commented?

Reply:
It was already commented